sdcm/cluster.py

```diff
@@ -3045,6 +3045,8 @@ def install_gemini(self, node):
     def node_setup(self, node, verbose=False, db_node_address=None, **kwargs):  # pylint: disable=unused-argument
         self.log.info('Setup in BaseLoaderSet')
         node.wait_ssh_up(verbose=verbose)
+        # add swap file
+        self.create_swap_file()
```

Comment:
Do we need to skip this for REUSE cluster? or checking if swap is configured before creating swap file?

Reply:
We should skip for REUSE, reuse assumes everything configured already.